src/electron/main.tsx

```diff
@@ -214,8 +213,22 @@ if (!handleSquirrelEvent()) {
           });
         }
       });
+      mainWindow.on('showMessageBox' , async (e, a) => {
+        await dialog.showMessageBox(mainWindow,a)
```

Comment:
please add reply for this function as well

Reply:
fixed